bindgen/Cargo.toml

```
@@ -1,11 +1,11 @@
 [package]
 name = "uniffi-bindgen-cpp"
-version = "0.7.4+v0.28.3"
+version = "0.8.0+v.0.29.4"
```

```suggestion
version = "0.8.0+v0.29.4"
```
there was an unneeded dot that we removed in the newer versions